client/src/main/java/io/split/engine/experiments/LocalhostSplitChangeFetcher.java

```diff
@@ -0,0 +1,41 @@
+package io.split.engine.experiments;
+
+import com.google.gson.Gson;
+import com.google.gson.stream.JsonReader;
+import io.split.client.dtos.SplitChange;
+import io.split.engine.common.FetchOptions;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.FileReader;
+
+public class LocalhostSplitChangeFetcher implements SplitChangeFetcher {
+
+    private static final Logger _log = LoggerFactory.getLogger(LocalhostSplitChangeFetcher.class);
+    private final File _file;
+
+    public LocalhostSplitChangeFetcher(String directory, String fileName){
+        _file = (directory.length() > 0) ?
+                new File(directory, fileName) :
+                new File(fileName);
+    }
+
+    @Override
+    public SplitChange fetch(long since, FetchOptions options) {
+        SplitChange splitChange = null;
+        try {
+            JsonReader jsonReader = new JsonReader(new FileReader(_file));
+
+            Gson gson = new Gson();
+            splitChange = gson.fromJson(jsonReader, SplitChange.class);
+        } catch (Exception e) {
+            _log.warn("There was no file named " + _file.getPath() + " found. " +
```

Comment:
maybe using String.format here instead of concatenating

Reply:
Added.